packages/docs/docs/subscriptions/subscription-extensions.md

```diff
@@ -116,7 +116,26 @@ The response for a deleted resource will contain:
 
 **_Executing only on delete_**
```

Comment:
Let's change this from bold text to 3-# header `### Subscription for "delete" events`